.github/workflows/releases.yml

```diff
@@ -20,14 +20,17 @@ permissions:
 
 jobs:
   generate-artifacts:
-    runs-on: ubuntu-20.04
+    runs-on: codebuild-soci-release-workflow-${{ github.run_id }}-${{ github.run_attempt }}-ubuntu-7.0-xlarge
     env:
       # Set during setup.
       RELEASE_TAG: ''
       DYNAMIC_BINARY_NAME: ''
       STATIC_BINARY_NAME: ''
     steps:
       - uses: actions/checkout@v4
+      - uses: actions/setup-go@v5
+        with:
+          go-version: ${{ env.GO_VERSION }}
```

Comment:
Do we still need Go here?

Reply:
It was required for the codebuild env, now since we are not including this workflow it is no longer required.